src/test/kotlin/uk/gov/justice/digital/hmpps/sentenceplan/integration/PopInfoControllerTest.kt

```diff
@@ -15,7 +15,7 @@ class PopInfoControllerTest : IntegrationTestBase() {
   fun `get pop info should return success`() {
     webTestClient.post().uri("/info/pop")
       .header("Content-Type", "application/json")
-      .headers(setAuthorisation(user = "Tom C", roles = listOf("ROLE_RISK_INTEGRATIONS_RO")))
+      .headers(setAuthorisation(user = "Tom C", roles = listOf("ROLE_SENTENCE_PLAN_READ")))
```

Comment:
The PopInfoController still has a PreAuthorize against ROLE_RISK_INTEGRATIONS_RO rather than ROLE_SENTENCE_PLAN_READ - should that be updated or removed?

Reply:
Interesting question - I myself was a bit unsure about how to handle this. It seems like, if I allowed a client with just the SENTENCE_PLAN_READ perms, they'd be able to make a kind-of proxied call through us to get this info - even though their client doesnt necessarily have the correct set of roles to call the endpoint at it's source.

For the moment, I just decided to leave it with the role enforced